File: Java/Functions/factorial.java
public class factorial {

    public static int factorial(int a) {
        int f = 1;

        for (int i = 1; i <= a; i++) {

            f = f * i;

        }
        return f;

    }

    public static void main(String[] args) {

        System.out.println(factorial(5));

    }

}

File: Cpp/Functions/factorial.cpp
#include <iostream>

using namespace std;

int factorial(int n) {
  int f = 1;
  for (int i = 1; i <= n; i++) {
    f *= i;
  }
  return f;
}

int main() {
  cout << factorial(5) << endl;
  return 0;
}
